### PR TITLE
added support for Nexys4DDR board in counter_test

### PR DESCRIPTION
**Pull request by cjearls**
Signed-off-by: Chandler Jearls <cjearls@vt.edu>

### CODE REVIEW

**Comment by mithro** [comment]
@cjearls - Just wanted to say, thank you for sending this pull request! @acomodi and @rw1nkler will help review it.

**Comment by cjearls** [comment]
Thanks, happy to help!

**Comment by acomodi** [comment]
Hi @cjearls. This looks good. I believe that there is something missing though to let the nexys4ddr test to run on CI.

Here you should add the mapping between the board identifier and the name that will be displayed in the docs: https://github.com/SymbiFlow/symbiflow-examples/blob/9a114c44c02c396b8e75812d8d21785ae3e3f84d/docs/collect_readmes.py#L5-L12.

While here you should add a line with the command to invoke to run this test, similarly to the other tests: https://github.com/SymbiFlow/symbiflow-examples/blob/master/xc7/counter_test/README.rst

IIRC, this is all that has to be done to enable the test in CI as well

**Comment by cjearls** [comment]
Ok, I've made those changes, let me know if there's anything else I can do!

File: docs/collect_readmes.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 full_name_lut = {
     'a35t': 'Arty 35T',
     'a100t': 'Arty 100T',
+    'nexys4DDR': 'Nexys 4 DDR',
```
Comment (acomodi) [inline docs/collect_readmes.py:8]: This should be `nexys4ddr` as expressed in the README, otherwise I believe a key-not-found error will be generated.

Comment (cjearls) [reply docs/collect_readmes.py]: Good catch! I fixed the typo.

**Comment by cjearls** [comment]
It looks like the tests automatically failed because there were no runners. Was I supposed to start runners for the CI when I submitted a PR?

**Comment by kgugala** [comment]
hi @cjearls the runners are back online. We have restarted the build. I see now they process regular nightly build, this PR is scheduled next.

**Comment by cjearls** [comment]
Thanks, I appreciate it!

**Comment by cjearls** [comment]
It looks like most of the tests passed, what's the best way to diagnose the checks that were unsuccessful?

**Comment by tcal-x** [comment]
```
03:27:43 | Writing Implementation FASM: top.fasm
03:27:43 | The entire flow of VPR took 1219.58 seconds.
03:27:44 | FASM extra: top_fasm_extra.fasm
03:27:44 | writing final fasm
03:27:47 | cd build/arty_100 && symbiflow_write_bitstream -d artix7 -f top.fasm -p xc7a100tcsg324-1 -b top.bit
03:27:47 | Writing bitstream ...
Error: The operation was canceled.
```

When I run on my laptop, I sometimes hit an out-of-memory error at this point, but only when building with the 200T part.

**Comment by tcal-x** [comment]
@cjearls , I've restarted the jobs.

**Comment by cjearls** [comment]
@acomodi @rw1nkler It looks like all the tests have passed, is there anything else that needs to be done for this to be accepted?

**Review by kgugala — APPROVED** [review]
LGTM

**Pull request merged**